src/platformdirs/unix.py

```diff
@@ -3,6 +3,26 @@
 
 from .api import PlatformDirsABC
 
+# Mapping between function name and relevant XDG var
+XDG_SUPPORT = {
+    "user_data_dir": "XDG_DATA_HOME",
+    "site_data_dir": "XDG_DATA_DIRS",
+    "user_config_dir": "XDG_CONFIG_HOME",
+    "site_config_dir": "XDG_CONFIG_DIRS",
+    "user_cache_dir": "XDG_CACHE_HOME",
+    "user_state_dir": "XDG_STATE_HOME",
+    "user_log_dir": "XDG_CACHE_HOME",  # Should be in XDG_STATE_HOME as per spec
+}
+# Mapping between function name and relevant XDG var
+XDG_DEFAULTS = {
```

Comment:
You define these but doesn't seem used. The scope of this PR is to change only macos, so not sure why we need to touch even this file.

Reply:
I wanted to document the spec compliance in a way such that it was usable in testing.

Reply:
I'd be -1 on this. IMHO the tests should not use such globals, because effectively you're no longer testing the content of this. A typo in this silently would be ignored. Let's not do this, the test can declare it's own copy of this, so we have some sanity check in between them.